layouts/partials/head.html

```diff
@@ -36,9 +36,6 @@
   })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
   </script>
 {{ end }}
-<link rel="preconnect" href="https://fonts.googleapis.com" />
-<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,500&display=fallback" />
-<link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=fallback" />
```

Comment:
See my other comment w.r.t. pre--fetching. I found an old PR where I added that (which also includes some comments that could be useful including "for reference");

- https://github.com/docker/docs/pull/11560

Reply:
Good callout - I have added `preload` to the RobotoFlex font, as it's critical and used on every page.

I am keen on keeping the `font-display: fallback` option over `swap`. They're very similar but `fallback` will **hide** text for up to 100ms until the web font is loaded, or after 100ms if the web font is not loaded it will display text in a fallback font. Whereas `swap` will display the fallback font immediately. The effective difference is that within the 100ms initial period, the text is either hidden or displayed with a fallback font. Hidden text reduces layout shifts, so I'm preferring this option.